tests/osaka/eip7692_eof_v1/eip7480_data_section/test_data_opcodes.py

```diff
@@ -81,6 +81,12 @@ def create_data_test(offset: int, datasize: int):
         pytest.param(0x5BFE, 0x5C00, id="short_23k"),
         pytest.param(0x5BE0, 0x5D00, id="exact_23k"),
         pytest.param(0x2345, 0x5C00, id="large_23k"),
+        pytest.param(2**16-1, 32, id="u16_max"),
```

Comment:
```suggestion
        pytest.param(2**16 - 1, 32, id="u16_max"),
```
Tox complains about no spaces between plus sign.